docs/tutorials/tabular_prediction/tabular-faq.md

```diff
@@ -164,6 +164,12 @@ The recommended workaround from the torch issue to suppress this warning is to s
 export OMP_NUM_THREADS=1
 ```
 
+### How to limit the number of cores AutoGluon will use
+Although it is generally recommended to let AutoGluon to use all the cores, you can limit it by setting the `ag_fit_args`:
+```
+predictor = TabularPredictor(...).fit(..., ag_fit_args={'num_cpus': NUM_CORES_YOU_WANT})
```

Comment:
Would be good to show an example of specifying it in hyperparameters, such as model 1 = default, model 2 = 1 cpus, model 3 = 4 cpus